source/issues-and-help.txt

```diff
@@ -0,0 +1,49 @@
+.. _pymongo-issues-help:
+
+=============
+Issues & Help
+=============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: suggestion, github, python
+
+We're glad to have such a vibrant community of {+driver-short+}
+users. We recommend seeking support for general
+questions through the `MongoDB Community Forums
+<https://www.mongodb.com/community/forums>`__.
```

Comment:
S: you can use the community forums directive here

```suggestion
questions through the :community-forum:`MongoDB Community Forums <tag/python/>`.
```

Reply:
cool!